### DISCUSSION TITLE
A discord to provide a common space for community interaction and asynchronous communication.

### DISCUSSION THREAD

**Discussion by brobbins2001** [polywock/Awesome-GPTs#19] [Ideas]
It would potentially be cool to set up a discord to allow GPT creators/users to engage in discourse about GPTs without the need to post here on this github or on the OpenAI forums. This would allow a greater separation of concerns for this github, by moving discussion away from the repo into a common space. I'd be willing to set this up if there is interest.

@polywock As the owner of this github, I'm very interested to hear your opinion on this

I'm interested in all opinions on this however.

**Comment by polywock** [comment]
I'll invite you to one I found. I don't think it's worth creating a separate one. 

https://t.co/N6OkZRqx4B

**Reply by brobbins2001** [reply]
Thank you, definitely not worht it if someone's already done it

**Reply by polywock** [reply]
By the way, I should mention that invite link is not mine, it's the same one I used to enter the Discord server.
